docs/core/runtime-config/garbage-collector.md

```diff
@@ -313,18 +313,19 @@ For more information about some of these settings, see the [Middle ground betwee
 
 ### Affinitize ranges
 
-- Specifies the list of processors to use for garbage collector threads.
+- Specifies the list of processors to use for garbage collector threads. This can either 
 - This setting is similar to [System.GC.HeapAffinitizeMask](#affinitize-mask), except it allows you to specify more than 64 processors.
-- For Windows operating systems, prefix the processor number or range with the corresponding [CPU group](/windows/win32/procthread/processor-groups), for example, "0:1-10,0:12,1:50-52,1:70".
+- For Windows operating systems, prefix the processor number or range with the corresponding [CPU group](/windows/win32/procthread/processor-groups), for example, "0:1-10,0:12,1:50-52,1:7". Note that you cannot use this config if you don't actually have more than 1 CPU group. You must use the [Affinitize mask](#affinitize-mask) setting. And the numbers you specify are within that group, which means it cannot be >= 64.
+- For Linux operating systems, since the [CPU group](/windows/win32/procthread/processor-groups) concept does not exist there you can use both this and the [Affinitize mask](#affinitize-mask) setting to specify the same ranges.
```

Comment:
> you can use both this and ...

Having read the info on the Windows list with group prefixes, this sounds as if Unix also used the prefixes. Maybe saying "you can you both the list of processors and ..." would be clearer?

Reply:
I thought mentioning that CPU group is a windows only thing made it obvious that linux does not use a group index  but I can make this clearer.

Reply:
I'll make this clearer - I meant you can use either this setting or the affinity mask setting.

Reply:
actually @gewarren's change made this clear.